src/lib/apis/tools/index.ts

```
@@ -34,6 +34,41 @@ export const createNewTool = async (token: string, tool: object) => {
 	return res;
 };
 
+export const loadToolByUrl = async (token: string = '', url: string) => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)